tests/unit/models/disposition-test.js

```diff
@@ -1,13 +1,50 @@
 import { module, test } from 'qunit';
 import { setupTest } from 'ember-qunit';
+import { STATUSCODES, STATECODES } from 'labs-zap-search/models/disposition';
+import { RECOMMENDATION_OPTIONSET_BY_PARTICIPANT_TYPE_LOOKUP } from 'labs-zap-search/controllers/my-projects/assignment/recommendations/add';
 
 module('Unit | Model | disposition', function(hooks) {
   setupTest(hooks);
 
   // Replace this with your real tests.
-  test('it exists', function(assert) {
+  test('it exists', async function(assert) {
     const store = this.owner.lookup('service:store');
     const model = store.createRecord('disposition', {});
+
     assert.ok(model);
   });
+
+  test('when participant submits/opts out, statuscode is "saved"', async function(assert) {
```

Comment:
Maybe the title should be when participant opts out/submits, statuscode becomes "saved"/"submitted"

Reply:
good catch!